File: frontend/app/src/main/java/sdu/msd/ui/createGroup/CreateGroupViewModel.java
package sdu.msd.ui.createGroup;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class CreateGroupViewModel extends ViewModel {
    private MutableLiveData<String> text = new MutableLiveData<>();

    public CreateGroupViewModel() {
        text.setValue("This is Fragment");
    }

    public LiveData<String> getText() {
        return text;
    }
}

File: frontend/app/src/main/java/sdu/msd/ui/createGroup/createGroupViewModel.kt
package sdu.msd.ui.createGroup

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class reateGroupViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is Fragment"
    }
    val text: LiveData<String> = _text
}